catalog/schematron/catalog-schematron-plugin/src/test/java/ddf/services/schematron/SchematronValidationServiceTest.java

```diff
@@ -166,6 +167,83 @@ public void testWithSpacesInPathToSchematronFile()
                 "dog_3leg_3paw.xml"));
     }
 
+    @Test
+    public void testWarningsAndErrorsAreSanitized()
+            throws ValidationException, IOException, SchematronInitializationException {
+        SchematronValidationService service = getService("dog_legs.sch", "dog_paws.sch");
+        try {
+            service.validate(getMetacard("dog_4leg_3paw.xml"));
+        } catch (SchematronValidationException ex) {
+            assertThat(service.getSchematronReport(), is(notNullValue()));
+            if (ex.getWarnings() != null) {
+                for (String warning : ex.getWarnings()) {
+                    assertThat(warning.equals(service.sanitize(warning)), is(true));
+                }
+            }
+            if (ex.getErrors() != null) {
+                for (String error : ex.getErrors()) {
+                    assertThat(error.equals(service.sanitize(error)), is(true));
+                }
+            }
+        }
+    }
+
+    @Test
+    public void testSanitizationWithNoRemoveableDelimiters()
+            throws SchematronInitializationException {
+        SchematronValidationService service = getService(false,
+                "doggy-namespace",
+                true,
+                "dog_legs.sch",
+                "dog_paws.sch");
+
+        String s = "abcdefghijk";
+        assertThat(s, is(service.sanitize(s)));
```

Comment:
I'd like to see another test, very similar, but with a space in the middle
`String s = "abcde fghijk";
assertThat(s, is(service.sanitize(s)));`